code/game/turfs/space/transit.dm

```diff
@@ -59,7 +59,8 @@
 	AM.newtonian_move(dir)
 
 /turf/open/space/transit/CanBuildHere()
-	return istype(get_area(src), /area/shuttle)
+	var/area/current = get_area(src)
+	return istype(current, /area/shuttle) && !istype(current,/area/shuttle/transit)
```

Comment:
I only tested blowing up midflight, the problem here is when you blow up before moving to default ignored area aka space. Gimme a sec i'll fix it up